CI/appveyor.after_success.ps1

```diff
@@ -3,16 +3,7 @@ if ("$Env:APPVEYOR_REPO_NAME" -ne "Mudlet/Mudlet") {
 }
 
 cd "$Env:APPVEYOR_BUILD_FOLDER\src\release"
-
-$Script:QtVersionRegex = [regex]'\\([\d\.]+)\\mingw'
```

Comment:
This is still a good change to have in the system isn't it? No need to revert it IMHO.

Reply:
Just makes it easy for the revert process though.